tests/helm/common_test.go

```diff
@@ -242,3 +242,40 @@ func GetPodSpec(object unstructured.Unstructured) (*corev1.PodSpec, error) {
 		return nil, nil
 	}
 }
+
+func TestNamespaceOverride(t *testing.T) {
+	valuesFilePath := path.Join(testDataDirectory, "everything-enabled.yaml")
+	namespaceOverride := "sumologic"
```

Comment:
Please set this to a value other than `sumologic`. Right now this test doesn't actually do anything.

Reply:
@swiatekm-sumo Updated to different value. Can you please approve PR checker run?